.github/workflows/quali.yml

```diff
@@ -1531,638 +1531,42 @@ jobs:
         run: python .quali/stop_reservation.py ${{ steps.reservation.outputs.identifier }}
 
   # Save reports
-  report-edgecore-eap102-1:
+  report:
+    if: "!cancelled()"
```

Comment:
@dunaev-opsfleet as far as I understand it in this case the report job will not run if one of the test jobs failed. Is this something we want?

Reply:
Is that so? Based on https://github.com/Telecominfraproject/wlan-testing/blob/master/.github/workflows/interop.yml#L767 and results from https://github.com/Telecominfraproject/wlan-testing/actions/runs/2348046194 report was starting as it's supposed

Reply:
You're right, I got a bit confused, subsequent jobs will run regardless of whether the previous job failed.